### DISCUSSION TITLE
KEYCLOAK (16.x and 17.x) : Compatibility with APIMAN 2.2.0

### DISCUSSION THREAD

**Discussion by JYassine** [apiman/apiman#1806] [Q&A]
Hello,

We would like to know if the versions **(16.x and 17.x)** are compatible and run with the version **2.2.0** of APIMAN ?

Thank you,
Sincerely.

**Comment by msavy** [comment]
Have you tried it out yet? How did it go?

15.1.1.Final is the latest bundled in the upstream, but it should work well with whatever version. But you'll need to test, of course. 

If you are supported by a downstream vendor (e.g. me), then feel free to contact them, as they will likely also be able to advise 🙌.
